src/test/java/org/vanilladb/core/storage/file/FileTest.java

```diff
@@ -231,7 +231,7 @@ public void testIsFileEmpty() {
 				fm.isFileEmpty(filename));
 		
 		// write 123 and 456 at block 0
-		BlockId blk = new BlockId(filename, 0);
+		BlockId blk = p1.append(filename);

```

Comment:
I think this line should remain the same (`BlockId blk = new BlockId(filename, 0);`) as in the original test case. Or, is there any specific reason that this line should use `append`?

Reply:
Fixed